Automattic/jetpack
# Plugin Search Hints: avoid PHP warnings

## Proposed changes:

Some other plugins may play with the plugins_api_result hook and not return the kind of results we're expecting. Let's avoid warnings in this case.

```
PHP Warning:  array_filter() expects parameter 1 to be array, null given in /wp-content/plugins/jetpack/modules/plugin-search.php on line 409

PHP Warning:  array_unshift() expects parameter 1 to be arr
```

### Other information:


## Jetpack product discussion

* p1685003148609169-slack-CDLH4C1UZ

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Test that feature hints still work:

* On a site that's connected to WordPress.com, go to Plugins > Add New
* Search for "cdn"
* See that Jetpack's Photon is suggested as the first result.
